core_perception/vision_darknet_detect/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ set(CMAKE_CXX_FLAGS "-O3 -g -Wall ${CMAKE_CXX_FLAGS}")
 
 AW_CHECK_CUDA()
 
-if (USE_CUDA)
+# if (USE_CUDA)
```

Comment:
We should keep the CUDA guard I would think. That will keep it compatible if we removed the build with CUDA flag. I think the more robust solution here would be to replace the ELSE() with ENDIF() such that non-cuda is always built but with-cuda is only built when the flag is provided.